echopype/convert/set_groups_ek60.py

```diff
@@ -308,15 +312,27 @@ def set_platform(self, NMEA_only=False) -> xr.Dataset:
                         },
                     },
                     coords={
-                        "ping_time": (
-                            ["ping_time"],
+                        "time2": (
+                            ["time2"],
+                            self.parser_obj.ping_time[ch],
+                            {
+                                "axis": "T",
+                                "long_name": "Timestamps for position datagrams",
+                                "standard_name": "time",
+                                "comment": "Time coordinate corresponding to platform sensors.",
```

Comment:
I think this comment should be "Time coordinate corresponding to platform motion and position sensors.". See @leewujung's initial comment/description in #656.

```suggestion
                                "comment": "Time coordinate corresponding to platform motion and position sensors.",
```